app/controllers/users_controller.rb

```diff
@@ -1,4 +1,28 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+    if @user.nil?
```

Comment:
cái này là anh check khi nhập id ko đúng cơ mà em. nếu nó không null "unless" thì in error ra làm gì em?